### PR TITLE
[URGENT] DEVELOPER-4501 Reverts DEVELOPER-4495

### PR DESCRIPTION
**Pull request by adelasofia**
Reverting DEVELOPER-4495 until we find another place for the tracking codes.

### CODE REVIEW

**Comment by adelasofia** [comment]
Do not merge yet, Jess will let us know if she wants this reverted

File: javascripts/build.js [hunk 1]
```diff
@@ -480,14 +480,14 @@ var DevNationLiveApp = (function (_super) {
...
...    </div>" : "\n                    <img width=\"640\" height=\"360\" src=\"../images/design/devnationlive_herographic_0.jpg\" alt=\"" + next.title + "\">\n                    ") + "\n                </div>\n                <div class=\"medium-10 columns event-chat\" data-chat=\"" + next.youtube_id + "\">\n                    " + (_this.getCookie('dn_live_' + next.offer_id) || !next.register ? "\n                    <iframe class=\"embedded-chat\" src=\"https://www.youtube.com/live_chat?v=" + next.youtube_id + "&embed_domain=" + window.location.href.replace(/http(s)?:\/\//, '').split('/')[0] + "\"></iframe>\n                    " : "\n                    <iframe class=\"session-reg\" src=\"" + _this.form + "?id=" + next.offer_id + "\"></iframe>\n                    ") + "\n                </div>\n            </div>\n            <div class=\"row\">\n                <div class=\"large-24 columns divider\">\n                    <p>Speaker: <strong>" + next.speaker + "</strong> \n                    " + (next.twitter_handle ? "\n                    (<a href=\"https://twitter.com/" + next.twitter_handle + "\" target=\"_blank\" class=\"external-link\"> @" + next.twitter_handle + "</a>)"
```
Comment (paulrobinson) [inline javascripts/build.js]: @adelasofia can you point me at what's changed on this line?

Comment (adelasofia) [reply javascripts/build.js]: @paulrobinson Same thing, but for some reason github didn't highlight the change
[see screenshot](https://www.dropbox.com/s/kv21odz7m3jctap/Screenshot%202017-11-01%2010.04.43.png?dl=0)

Comment (LightGuard) [reply javascripts/build.js]: @paulrobinson This file is a bunch of compiled typescript. You'll want to see the changes down below.

**Pull request merged**